notes.xsl

```diff
@@ -67,6 +67,9 @@
 		<xsl:when test="@id = 'detach-shorthand'">
 			As the <code>.<xsl:value-of select="@data-event"/>()</code> method is just a shorthand for <code>.on( "<xsl:value-of select="@data-event"/>", handler )</code>, detaching is possible using <code>.off( "<xsl:value-of select="@data-event"/>" )</code>.
 		</xsl:when>
+		<xsl:when test="@id = 'global-ajax-event'">
+			Starting with jQuery 1.9, all the handlers for the <a href="/category/ajax/global-ajax-event-handlers/">jQuery global Ajax events</a>, including those added with the <code><xsl:value-of select="@data-title"/></code> method, must be attached to <code>document</code>.
```

Comment:
Ah, I see. I think what you have is fine then. We typically use "As of jQuery x.x" instead of "Starting with x.x." Would you mind changing that phrase?